src/main/java/com/takipi/oss/storage/resources/fs/multifetcher/SimpleStopWatch.java

```diff
@@ -0,0 +1,18 @@
+package com.takipi.oss.storage.resources.fs.multifetcher;
+
+class SimpleStopWatch
```

Comment:
why stopwatch and not guava stopwatch?

Reply:
Guava stopwatch is heavier and more complex. I wanted something as simple as possible.